presto-benchto-benchmarks/README.md

```diff
@@ -21,6 +21,7 @@ features. Therefore benchmarks are organized in suites, like:
 * Presto already installed on the target environment
 * Basic understanding of Benchto [components and architecture](https://github.com/prestodb/benchto)
 * Benchto service [configured and running](https://github.com/prestodb/benchto/tree/master/benchto-service)
+* Benchto driver JAR [built](https://github.com/prestodb/benchto/tree/master/benchto-driver)
```

Comment:
shouldn't we rather suggest downloading a release from maven central?

or even a copy-pastable wget to do this?

or, maybe, just as you can invoke _any_ plugin with maven (without configuring it first), you can `./mvnw ....` to download and invoke the benchto driver in one shot?